homeassistant/components/knx/__init__.py

```diff
@@ -361,9 +402,7 @@ async def connection_state_changed_cb(self, state: XknxConnectionState) -> None:
 
     def register_callback(self) -> TelegramQueue.Callback:
         """Register callback within XKNX TelegramQueue."""
-        address_filters = list(
-            map(AddressFilter, self.config[DOMAIN][CONF_KNX_EVENT_FILTER])
-        )
+        address_filters = list(map(AddressFilter, self.config[CONF_KNX_EVENT_FILTER]))
```

Comment:
This is more readable 
```suggestion
        address_filters = [
            AddressFilter(filter) for filter in self.config[CONF_KNX_EVENT_FILTER]
        ]
```